packages/babel-cli/src/babel/options.js

```diff
@@ -73,7 +73,7 @@ commander.option(
   "Do not include superfluous whitespace characters and line terminators.",
   booleanify,
 );
-commander.option("--minified", "save as much bytes when printing [true|false]");
+commander.option("--minified", "Save as many bytes when printing. [true|false]");
```

Comment:
```suggestion
commander.option("--minified [true|false]", "Save as many bytes when printing.");
```
To be consistent with other descriptions.